cmd/res/profiles/hikvision-profile.yaml.example

```diff
@@ -0,0 +1,18 @@
+name: "hikvision-profile"
+manufacturer:  "HIKVISION"
+model: "DFI6256TE"
+labels:
+  - "onvif"
+description: "EdgeX device profile for HIKVISION ONVIF-compliant IP camera."
+
+deviceResources:
+  - name: "Users"
```

Comment:
Why only one resource??

Reply:
It's an example profile, user can create their own.

Reply:
Is see `camera.yaml` as the example. By naming this one `hikvision`, one would expect it to be fully flushed out for `Hikvision`. Recommend removing it if not willing to make it a complete  `hikvision` profile.

Reply:
Got it, removed the example profile.